web/views/auth_view.ex

```diff
@@ -0,0 +1,27 @@
+defmodule CodeCorps.AuthView do
+  use CodeCorps.Web, :view
+
+  def render("show.json", %{token: token, user_id: user_id}) do
+    %{
+      token: token,
+      user_id: user_id,
```

Comment:
Right now it is needed, but we can refactor the Ember side to pull it from the JWT if we'd like.